Python/flowgraph.c

```diff
@@ -515,22 +515,6 @@ no_redundant_jumps(cfg_builder *g) {
     }
     return true;
 }
-
-static bool
-all_exits_have_lineno(basicblock *entryblock) {
```

Comment:
Could you comment out rather than delete?

Reply:
I'd rather delete it, otherwise it just rots.

We do have version control, if you need it back.

Reply:
> If it doesn't get deleted, compiler may raise warnings

But not if it's commented out.